config.m4

```diff
@@ -1,6 +1,6 @@
 dnl config.m4 for extension mongodb
-PHP_ARG_ENABLE(mongodb, whether to enable mongodb support,
-[  --enable-mongodb           Enable mongodb support])
+PHP_ARG_ENABLE(mongodb, whether to enable MongoDB support,
+[  --enable-mongodb        Enable MongoDB support])
```

Comment:
These should really use `AC_HELP_STRING`. Would you like to re-work this commit? Otherwise, pull it out entirely and I'll address this in my original PR.

Reply:
``AC_HELP_STRING`` is something I had not encountered before. I don't see any other PHP extensions using it. It was actually working against me, as originally I wanted to indent all the dependent settings with two extra spaces, but m4 seems to remove beginning white space from the string...

Reply:
I'm not surprised that other PHP extensions don't use it, but I think it's worth using in our case since we have so many options and their help text often runs multiple lines. See: https://autotools.io/autoconf/arguments.html#autoconf.arguments.helpstrings

There are some other benefits aside from formatting.